### DISCUSSION TITLE
Excercise 4 - Aufgabe 3: Zeichen oder Buchstaben?

### DISCUSSION THREAD

**Discussion by Foxglovve** [irgroup-classrooms/dis08-2021#40] [Q&A]
Hallo, 

ich habe eine Frage zur Formulierung von oben genannter Aufgabe:
"Count the number of lines that contain a chat message with at least 20 characters in all .csv files. Hint: Check on the correct pattern with grep."
Sind hier mit characters tatsächlich nur Buchstaben gemeint oder Zeichen an sich?

Danke und viele Grüße
Marion Bachmann

**Comment by HaakFab** [comment]
Hallo, 
Characters sind ALLE Zeichen, also quasi in einer Reg Ex der Punkt Operator.
VG,
Fabian Haak

**Reply by Foxglovve** [reply]
Ok, dankeschön!
